src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -797,14 +799,14 @@ export class SearchInterface extends RootComponent implements IComponentBindings
     if (quickviewRef) {
       const quickviews = this.getComponents<any>(quickviewRef.ID);
       if (args.value != '') {
-        const quickviewsPartition = _.partition(quickviews, quickview => quickview.getHashId() == args.value);
+        const quickviewsPartition = partition(quickviews, quickview => quickview.getHashId() == args.value);
```

Comment:
One nice thing you could do here is to use destructuring, creating two variable names that explain the two sides of the partition. e.g.

```
[quickViewsMatchingHash, quickViewsNotMatchingHash] = partition(...)
```